packages/store-cache/src/utils.ts

```diff
@@ -116,31 +119,37 @@ export function subscribe<C extends StoreConfig = StoreConfig, T extends keyof C
     // Transform the writes into the expected format
     for (const update of write.set ?? []) {
       // The first tuple element is always the table name
-      const table = update.key[0];
-      if (typeof table !== "string") {
-        console.warn("store-cache: Expected first tuple element to be the table id, ignoring set operation:", set);
+      const [namespace, table] = update.key;
+      if (typeof namespace !== "string" || typeof table !== "string") {
+        console.warn(
+          "store-cache: Expected first tuple elements to be namespace and table, ignoring set operation:",
+          set
```

Comment:
where does `set` come from? should this be `update`?

Reply:
it comes from calling `set` on the tuple database (kept the names that `tuple-database` is using here, which also kinda corresponds to the naming used in `Store`)